vpt/processors/gaze_detector/__init__.py

```diff
@@ -163,10 +163,11 @@ def detect_marks(self, image_np):
 
 class GazeDetector(ProcessorBase[np.ndarray]):
     '''Detects if the user is looking at the screen or not'''
-    subj = Subject()
+    __subj: Subject
```

Comment:
You called it `__subj` here, but used `_subj` in the function body, so they are different fields. Also, shouldn't the type be `Observable`?